File: 1561_Maximum_Number_of_Coins_You_Can_Get/solution.c
int comp(const void* a, const void* b) { return (*(int*)a - *(int*)b); }
int maxCoins(int* piles, int pilesSize) {
    qsort(piles, pilesSize, sizeof(piles[0]), comp);
    int result = 0;
    for (int i = 0; i < pilesSize / 3; i++) {
        result += piles[pilesSize - (2 * i + 2)];
    }
    return result;
}

File: 1561_Maximum_Number_of_Coins_You_Can_Get/solution.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        piles.sort(reverse=True)
        n = len(piles) // 3
        result = 0
        for i in range(n):
            result += piles[2 * i + 1]

        return result